resources/main.js

```diff
@@ -48,6 +49,23 @@ function copyToClipboard(temptext) {
         tempItem.focus();
         tempItem.select();
         document.execCommand('copy');
-        mw.notify( mw.message( 'scratch-confirmaccount-click-copy-alert', { autoHide: true }, {autoHideSeconds: 5}) ); // Use an i18n message to send a notification
         tempItem.parentElement.removeChild(tempItem);
       }
+
+
+$(function () {
+    const elem = document.getElementsByName("scratchusername")[0];
+    if (!elem) return;
+
+    elem.onblur = function() {
+	var currentname = elem.value || "";
+        var usernameblock = new OO.ui.infuse(elem.parentElement.parentElement.parentElement.parentElement);
```

Comment:
Would it be possible to use `.closest` instead of this? This seems janky and likely to break with some OOUI update.